halodi-controller-simulation-api/build.gradle

```diff
@@ -1,12 +1,13 @@
 buildscript {
     repositories {
        mavenCentral()
+       maven { url "https://plugins.gradle.org/m2/" }
        mavenLocal()
        jcenter()
     }
     dependencies {
         classpath 'com.github.jengelman.gradle.plugins:shadow:5.0.0'
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.+'
+        classpath 'net.saliman:gradle-properties-plugin:1.5.1'
```

Comment:
Where is this used? I think the rest of the code is builtin

Reply:
It is used to choose between gradle.properties files, so that we can use a gradle-release.properties.